app/javascript/packs/application.js

```diff
@@ -13,7 +13,14 @@ Turbolinks.start()
 // ActiveStorage.start()
 
 
-document.addEventListener("turbolinks:load", function() {
+document.addEventListener("turbolinks:load", function(event) {
+
+  if (typeof gtag === 'function') {
+    gtag('config', '<%= Rails.application.credentials.dig(:google_analytics) %>', {
```

Comment:
I didn't use Rails credentials before. Shouldn't there be a `.enc` file also?

Reply:
` bin/rails credentials:edit` didn't make the change in `credentials.yml.enc`, fixing that, good catch!